stdlib.go

```diff
@@ -135,16 +143,17 @@ const (
 )
 
 var (
-	logRegexp = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpFile + logRegexpMsg)
+	StdlibLogRegexpDateTimeFileMsg = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpFile + logRegexpMsg)
+	StdlibLogRegexpDateTimeMsg     = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpMsg)
 )
 
-func subexps(line []byte) map[string]string {
-	m := logRegexp.FindSubmatch(line)
-	if len(m) < len(logRegexp.SubexpNames()) {
+func (a StdlibAdapter) subexps(line []byte) map[string]string {
+	m := a.logRegexp.FindSubmatch(line)
```

Comment:
Should we be using FindStringSubmatch here instead?

Reply:
We can use `FindStringSubmatch` here, I'm not aware of any differences.